cypress/e2e/smoke/single_student_canvas_test.js

```diff
@@ -13,6 +24,17 @@ let imageToolTile = new ImageToolTile;
 let drawToolTile = new DrawToolTile;
 let textToolTile = new TextToolTile;
 let tableToolTile = new TableToolTile;
+let questionToolTile = new QuestionToolTile;
+const dialog = new Dialog();
+const arrowAnnotation = new ArrowAnnotation();
+const dataCardToolTile = new DataCardToolTile();
+const dataflowToolTile = new DataflowToolTile();
+const diagramToolTile = new DiagramToolTile();
+const simulatorTile = new SimulatorTile();
+const numberlineToolTile = new NumberlineToolTile();
+const expressionToolTile = new ExpressionToolTile();
+const xyPlotToolTile = new XYPlotToolTile();
+const barGraphTile = new BarGraphTile();
```

Comment:
It's a little odd to have some of these `let` and some of them `const`. I'd make them all the same.